### PR TITLE
Add client translations for Bed

### PR DESCRIPTION
**Pull request by L3ice**
Added the use of the built-in translation from mcpe.

### CODE REVIEW

**Review by robske110 — CHANGES_REQUESTED** [review]
You also need to include this in the PM translations, in order to not break force-language

**Review by SOF3 — CHANGES_REQUESTED** [review]
As @robske110 mentioned.

File: src/pocketmine/block/Bed.php [hunk 1]
```diff
@@ -86,15 +87,15 @@ public function onActivate(Item $item, Player $player = null){
 				$b = $blockWest;
 			}else{
 				if($player instanceof Player){
-					$player->sendMessage(TextFormat::GRAY . "This bed is incomplete");
+					$player->sendMessage(new TranslationContainer(TextFormat::GRAY . "%tile.bed.notValid"));
```
Comment (dktapps) [inline src/pocketmine/block/Bed.php]: In the game files, this is actually "Your home bed was missing or obstructed". I don't think this is correct.

**Review by dktapps — DISMISSED** [review]

**Comment by robske110** [comment]
I qoute:
@robske110 suggested changes ? days ago
You also need to include this in the PM translations, in order to not break force-language
 @SOF3	SOF3 requested changes 17 days ago
As @robske110 mentioned.

**Comment by L3ice** [comment]
I think they know that...

**Comment by dktapps** [comment]
Merged changes in the referenced commit.

**Pull request closed without merging**